modules/docker/Makefile.build

```diff
@@ -0,0 +1,12 @@
+.PHONY: docker\:build
+## Use DOCKER_IMAGE_NAME envvar to specify docker image with tags
+## Build docker image
+docker\:build: $(DOCKER)
```

Comment:
This is not true.
I tested call of deps targets. It works in both cases with or without \:
But on Linux $(CMD) as deps never run